harbor/tests/test_unit.py

```diff
@@ -52,6 +52,16 @@ def test_submit_disk_metrics(aggregator, harbor_check, harbor_api):
     aggregator.assert_metric('harbor.disk.total', 1e6, tags=tags)
 
 
+@pytest.mark.usefixture("patch_requests")
+def test_submit_read_only_status(aggregator, harbor_check, harbor_api):
+    if harbor_api.harbor_version >= VERSION_1_5:
+        tags = ['tag1:val1', 'tag2']
+        harbor_check._submit_read_only_status(harbor_api, tags)
+        aggregator.assert_metric('harbor.registry.read_only', False, tags=tags)
+    else:
+        None
```

Comment:
```suggestion
@pytest.mark.usefixture("patch_requests")
@pytest.mark.skipif(harbor_api.harbor_version < VERSION_1_5, reason="The registry.read_only metric is only submitted for Harbor 1.5+")
def test_submit_read_only_status(aggregator, harbor_check, harbor_api):
    tags = ['tag1:val1', 'tag2']
    harbor_check._submit_read_only_status(harbor_api, tags)
    aggregator.assert_metric('harbor.registry.read_only', False, tags=tags)
```

Reply:
Thank @FlorianVeaux. I had to update this a bit due to the following error. I'm not very familiar with this, so I can update if there's a better way to handle this.

```tests/test_unit.py:56: in <module>
    @pytest.mark.skipif(harbor_api.harbor_version < VERSION_1_5, reason="The registry.read_only metric is only submitted for Harbor 1.5+")
E   NameError: name 'harbor_api' is not defined
```

I also updated `metadata.csv`